.github/actions/deploy/deploy.py

```
@@ -0,0 +1,836 @@
+        print("🌐 Setting up port forwarding...")
+
+        # Check if service exists
+        try:
+            svc_output = self.run_command(
+                f"kubectl get service mlflow -n {self.args.namespace} -o yaml",
+                capture_output=True
+            )
+            if not svc_output:
+                print("❌ MLflow service not found")
+                return
+        except:
```

Ruff (0.14.13)

Do not use bare `except`

(E722)